tests/test_fixtures/great_expectations_no_block_no_fluent_datasources_stats_enabled.yml

```diff
@@ -0,0 +1,56 @@
+# This is a basic configuration for testing.
+# It has comments that should be preserved.
+config_version: 3
+datasources:
```

Comment:
Are these fixtures used at the moment?

Reply:
@Shinnnyshinshin Not yet -- preparation for next PR.

Reply:
If not, just curious why we are adding at this now

Reply:
They will be used in the next pull request.